### PR TITLE
Hvqfn 24

### PR DESCRIPTION
**Pull request by jneiva08**
Created footprint for HVQFN 24

Datasheet: https://www.nxp.com/docs/en/package-information/SOT616-3.pdf
and https://www.nxp.com/docs/en/data-sheet/NHS3100.pdf

### CODE REVIEW

**Comment by jneiva08** [comment]
When try to generate the footprint with the last version  shows the following error:

`$ ./ipc_noLead_generator.py size_definitions/qfn/hvqfn.yaml Building footprint for parameter set: HVQFN-24-1EP_4x4mm_P0.5mm_EP2.5x2.5mm
The use of manual heel reduction is deprecated. It is automatically calculated from the minimum EP to pad clearance (ipc config file)`

what can do to fix-it?

**Comment by chschlue** [comment]
Remove the `heel_reduction` parameter.

**Comment by jneiva08** [comment]
thanks for the help, it worked

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/hvqfn.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+HVQFN-24-1EP_4x4mm_P0.5mm_EP2.5x2.5mm:
```
Comment (chschlue) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/hvqfn.yaml]: Should be EP2.6x2.6mm

**Comment by chschlue** [comment]
FP merged.
Size definition still needs to be corrected.

**Comment by jneiva08** [comment]
@chschlue , Only needs to change EP size on name? and not the actual size?
It's recommended to make new request to update in footprint?

**Comment by chschlue** [comment]
EP size is already 2.6x2.6, just the name of the definition here is wrong.

**Comment by jneiva08** [comment]
tks

**Pull request merged**